File: CodeGeneratorGUI/DataAccessLayer/Models/ProjectMetadata.cs
using System;

namespace DataAccessLayer.Dto
{
    public class ProjectMetadata
    {
      public string Name { get; set; }
      public string Description { get; set; }
      public string Path { get; set; }
      public List<ModelMetadata> Models { get; set; }
      public List<FormMetadata> Forms { get; set; }

    }
}

File: CodeGeneratorGUI/ReactRedux/src/models/ProjectMetadata.ts
import { ModelMetadata } from "./ModelMetadata";
import { FormMetadata } from "./FormMetadata";
export interface ProjectMetadata {
  name:string;
  description:string;
  path:string;
  models:ModelMetadata[];
  forms:FormMetadata[];
}

export const initProjectMetadata = {
  name:'',
  description:'',
  path:'',
  models:[],
  forms:[],

}
